ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -116,13 +119,6 @@ import qualified System.Random                       as Random
 import           System.Random                       (RandomGen)
 import           Text.ParserCombinators.ReadP        (readP_to_S)
 
-
--- See Note [Guidelines For Using CPP In GHCIDE Import Statements]
```

Comment:
removing 9.2 compat code removed a lot of import CPP, making many of these comments irrelevant.